### PR TITLE
Added Jovo's proposed schedule spring 2023

### PR DESCRIPTION
**Pull request by bdpedigo**

### CODE REVIEW

**Review by jdey4 — DISMISSED** [review]

File: content/about/agreements.md [hunk 1]
```diff
@@ -236,6 +236,22 @@ Travel plans (when you are going to be out of lab for some period of time) shoul
 
 While these agreements are a living, breathing document, changing them costs energy for everybody, and can lead to an experience of instability.  Consider the costs upon proposing a modification. If you propose one, you are responsible for getting a majority of Neurodata members to approve the change.
 
+## Schedule
+The following schedule is planned beginning Spring semester 2023:
+- Jovo plans to be on campus (at least) on Mondays and Tuesdays all day. This should facillitate time for quick things one might need from Jovo.
+- Lab meetings will be 12-1pm on Mondays.
```
Comment (AliceQingyangWang) [inline content/about/agreements.md]:
```suggestion
- Lab meetings will be 12-1pm on Mondays. Sign-up sheet may be found [here](https://docs.google.com/spreadsheets/d/1pysmRprg6r2ToyddZhNx3AqCHm2aeHUy949h4KkcQso/edit?usp=sharing). 
```

**Comment by jshinm** [comment]
Sorry @bdpedigo! I just read your instruction. I reverted my last change 🙏

**Review by tathey1 — APPROVED** [review]
looks good to me.

**Comment by jshinm** [comment]
@bdpedigo @tathey1 Should this be merged now?

**Comment by bdpedigo** [comment]
sure!

**Pull request merged**